src/libopensc/pkcs15-idprime.c

```diff
@@ -98,7 +98,8 @@ static int sc_pkcs15emu_idprime_init(sc_pkcs15_card_t *p15card)
 	pin_info.attrs.pin.max_length    = 16;
 	pin_info.tries_left    = -1;
 
-	if (card->type == SC_CARD_TYPE_IDPRIME_V3) {
+	if (card->type == SC_CARD_TYPE_IDPRIME_V3 ||
+	card->type == SC_CARD_TYPE_IDPRIME_V4) {
```

Comment:
Could you fix the indent, please?

Reply:
Second expression should be aligned with the enclosed `if` block, correct? The Linux kernel style guide doesn't seem to have a definite answer on how to align multiple expressions.